ios/FluentUI/Vnext/List/HeaderTokens.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// HeaderFooter styles.
+@objc public enum HeaderStyle: Int, CaseIterable {
```

Comment:
We need to be careful about how we rename some of these enums, particularly regarding how they are projected into Objective-C. We need to keep our three-letter `MSF` prefix for anything Objective-C compatible, because that's our only differentiation from other symbols (there is no proper namespacing in ObjC).

Reply:
Added them back in!